message/validation/validation.go

```diff
@@ -88,13 +89,14 @@ type messageValidator struct {
 }
 
 // NewMessageValidator returns a new MessageValidator with the given network configuration and options.
-func NewMessageValidator(netCfg networkconfig.NetworkConfig, opts ...Option) MessageValidator {
+func NewMessageValidator(netCfg networkconfig.NetworkConfig, limiterCacheSize int, opts ...Option) MessageValidator {
 	mv := &messageValidator{
 		logger:                  zap.NewNop(),
 		metrics:                 &nopMetrics{},
 		netCfg:                  netCfg,
 		operatorIDToPubkeyCache: hashmap.New[spectypes.OperatorID, *rsa.PublicKey](),
 		validationLocks:         make(map[spectypes.MessageID]*sync.Mutex),
+		peerRateLimiter:         NewPeerRateLimiter(20, 300000, limiterCacheSize, 1*time.Minute),
```

Comment:
This `20` is the rejection limit value, right? As we discussed, I think this value should be higher. The maximum no. of rejections is 219, but it should be a little below that due to delay, right?